dot/parachain/types/types.go

```diff
@@ -654,3 +655,13 @@ type UncheckedSignedAvailabilityBitfield struct {
 	// The signature by the validator of the signed payload.
 	Signature ValidatorSignature `scale:"3"`
 }
+
+// Subsystem is an interface for subsystems to be registered with the overseer.
+type Subsystem interface {
```

Comment:
There are two Subsystem interfaces, one here and one in https://github.com/ChainSafe/gossamer/blob/0dcd13fc6c7d6e26c8fbd5a3a7c238aefe13d139/dot/parachain/overseer/types.go#L13

Let's use just one of these two. Let try to use overseer.Subsystem, if possible

Reply:
Yes, there should only be one.  I created the one in `types` so that it's outside the `overseer` package since a subsystem can be implemented without the awareness of the overseer.  With it in types we can create a test overseer in the subsystem tests without creating a cycle import. 

So I'd like to just use  `types.Subsystem`